packages/salesforcedx-vscode-core/src/commands/templates/apexGenerateBasicTestClass.ts

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
What's the difference between a basic test class and the original?

Reply:
the basic UT just is an apex test class outline, the standard one has a large comment at the top describing how to test, what should be test, etc...

Reply:
@WillieRuemmele is there any function difference between the two for the work you are currently doing. If not, please check with Ananya to see if we really need two variations of an Apex unit test template.

If both are needed then I would suggest that instead of adding a distinct command, that the current command to create a unit test be modified to prompt the user for the template name.

Reply:
Another option would be to reduce the verbosity of the current template, possible adding pointers to web docs/resources that cover all the details presented in the comment block.

Reply:
I'll look at modifying the current command to create this basic template. I wasn't sure if it would be easier to do that, or create this new command. It should ideally be invisible to customers to call/use and should only be used by E4D for now

Reply:
@WillieRuemmele what is the reason for the new template?

Reply:
for when Test Case Generation generates a new unit test file completely, the original template had a large comment at the top, and an empty test method that wasn't required.

https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001gtJt8YAE/view

Reply:
@peternhale I've put up this draft https://github.com/forcedotcom/salesforcedx-vscode/pull/5331 as an alternative, I believe it'll achieve the same results for E4D, and the same UX as the existing command